lms/djangoapps/grades/signals/handlers.py

```diff
@@ -56,13 +58,14 @@ def submissions_score_set_handler(sender, **kwargs):  # pylint: disable=unused-a
     if user is None:
         return
 
-    PROBLEM_WEIGHTED_SCORE_CHANGED.send(
+    PROBLEM_WEIGHTED_SCORE_CHANGED.send_robust(
```

Comment:
If this PR can go without this change, I would support putting it off until next week when we can do a deep dive on what signal handlers this affects and what test coverage looks like for those handlers. I don't like the idea of only finding problems with LTI because of a change we made in an adjacent app unless the problems arise via a legitimate integration test (as opposed to a test that relies upon a signal that incidentally also gets handled in LTI). That said, if the alternative is to send signals to apps without any integration tests in place, then I think that keeping the `send`s non-robust for now does make sense.